doc/Instrumentation.md

```diff
@@ -23,15 +23,15 @@ wizeng --monitors=loops,coverage{code=instr} test.wasm
 
 ## Options
 
-When no optional arguments are provided, most monitors display only summaries. The `code` argument will print out instrumentation infromation alongside bytecode, where possible. The available monitors and their options are as follows:
+When no optional arguments are provided, most monitors display only summaries. The `code` argument will print out instrumentation infromation alongside bytecode, where possible. The `csv` argument outputs the monitor data in a csv format to the terminal. The available monitors and their options are as follows:
 
 | Monitor    | Options                                    | Description                                                                                         |
 |------------|--------------------------------------------|-----------------------------------------------------------------------------------------------------|
-| `loops`    | `{c\|code}`                                | Analyzes loop blocks and their execution behaviour.                                                 |
+| `loops`    | `{c\|code}` `{csv}`                        | Analyzes loop blocks and their execution behaviour.                                                 |
```

Comment:
The `-csv` option is external to this, so this might be a misleading. I've been meaning to think about cleaning up how all the options are parsed and now might be a good time to think about a good strategy for all the monitors' options.